test/e2e/resource_validation_test.go

```diff
@@ -0,0 +1,226 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+package e2e
+
+import (
+	"errors"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/types"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"reflect"
+
+	clusterv1beta1 "go.goms.io/fleet/apis/cluster/v1beta1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Resource validation tests for Member Cluster", func() {
```

Comment:
can you split the tests into two, one is to allow create and the other one is to deny to improve the readability.